plugins/BEdita/Core/src/ORM/Locator/TableLocator.php

```diff
@@ -37,10 +37,6 @@ class TableLocator extends CakeLocator
      */
     protected function _getClassName($alias, array $options = [])
     {
-        if (empty($options['className'])) {
-            $options['className'] = Inflector::camelize($alias);
-        }
-
```

Comment:
those lines were removed because they are useless, same code is called in parent class
https://api.cakephp.org/3.2/source-class-Cake.ORM.Locator.TableLocator.html#152